user/troubleshooting/installation-troubleshooting.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: doc
+title: Installation Troubleshooting
+permalink: /doc/installation-troubleshooting/
+---
+# Installation Troubleshooting #
+
+## "An unknown error has occured" error during installation ##
+
+Some people have encountered this error message when trying to install Qubes on drives that already have data on them. 
+The solution is to exit the installer, wipe all data or delete all partitions, then restart the Qubes installation. 
+
+## Trouble installing from USB stick ##
+
+If you are facing issues when booting using UEFI mode, see the [UEFI troubleshooting guide](/doc/uefi-troubleshooting/). 
+
+There are a variety of other problems that could arise when using a USB installation medium, and some of the issues can be fixed by doing one or more of the following:
+
+* **Use a different USB drive:** 
+If possible, try several drives of different sizes and formats. 
+This determines whether the problem stems from the flash drive or Qubes installer. 
+Some laptops cannot read from an external boot device larger than 8GB. 
+If you encounter a black screen when performing an installation from a USB stick, ensure you are using a USB drive less than 8GB, or a partition on that USB less than 8GB and of format FAT32.
```

Comment:
In this case it would be a problem. Installation image is over 4GB, so it's unlikely to fit on a smaller USB. If the laptop is unable to boot from bigger USB, it may be too old to run Qubes at all.